src/curtis/__main__.py

```diff
@@ -95,13 +112,31 @@ def browse_unseen_issues(cfg, age):
     help="Issues with a trend ratio below this threshold are ignored",
     show_default=True,
 )
+@click.option(
```

Comment:
You can create a `with_filters` decorator that just apply the two decorators on the decorated function